frontend/integration-tests/views/metalkube/dashboards.view.ts

```diff
@@ -0,0 +1,127 @@
+import { browser, element, by , $$ } from 'protractor';
+import { waitForNone } from '../../protractor.conf';
+
+export const untilNoLoadersPresent = waitForNone($$('.co-m-loader'));
+export const isLoaded = () => browser.wait(untilNoLoadersPresent).then(() => browser.sleep(2000));
+
+// Usage example:
+// await dashboardView.inventoryItemLabel(0).getText()
+export function inventoryItemLabel(rowNumber) {
+  return element(by.xpath(`(//div[@class="kubevirt-inventory__row-title"])[${rowNumber + 1}]`));
+}
+
+// Usage example:
+// await dashboardView.inventoryUpCounter(0).getText()
+export function inventoryUpCounter(rowNumber) {
+  // Explaining the xpath:
+  // There is an array of 6 kubevirt-inventory__row-status items (one for each line in the inventory).
+  // Then we look for a kubevirt-inventory__row-status-item which has a kubevirt-inventory__row-status-item-icon--ok
+  // under it, and go __back up__ to the parent (with "..") and find the span under it. This gives us
+  // the counter *next to* the kubevirt-inventory__row-status-item-icon--ok.
+  return element(by.xpath(`(//div[@class="kubevirt-inventory__row-status"])[${rowNumber + 1}]` +
+    '/div[contains(@class, "kubevirt-inventory__row-status-item")]' +
+    '/span[contains(@class, "kubevirt-inventory__row-status-item-icon--ok")]' +
+    '/../span[contains(@class, "kubevirt-inventory__row-status-item-text")]'));
+}
+
+// Usage example:
+// await dashboardView.inventoryDownCounter(0).getText()
+export function inventoryDownCounter(rowNumber) {
+  // Similar xpath explanation to the one in inventoryUpCounter(), but now we're finding the counter
+  // next to the kubevirt-inventory__row-status-item-icon--error.
+  return element(by.xpath(`(//div[@class="kubevirt-inventory__row-status"])[${rowNumber + 1}]` +
+    '/div[contains(@class, "kubevirt-inventory__row-status-item")]' +
+    '/span[contains(@class, "kubevirt-inventory__row-status-item-icon--error")]' +
+    '/../span[contains(@class, "kubevirt-inventory__row-status-item-text")]'));
+}
+
+// substrings to identify the rows in the inventory card
+export const INVENTORY_NODES = ' Nodes';
+export const INVENTORY_HOSTS = ' Hosts';
+export const INVENTORY_PVCS = ' PVCs';
+export const INVENTORY_PODS = ' Pods';
+export const INVENTORY_VMS = ' VMs';
+export const INVENTORY_DISKS = ' Disks';
+
+// Usage example:
+// const rowNumber = await dashboardView.inventoryRow(dashboardView.INVENTORY_HOSTS);
+export const inventoryRow = async(substrInventory) => {
+  const elements = await $$('.kubevirt-inventory__row-title');
+  let i; // not using a forEach loop because you can't break or return from it
+  for (i = 0; i < elements.length; i++) {
+    const inventoryLabel = await elements[i].getText();
+    if (inventoryLabel.indexOf(substrInventory) > -1) {
+      return new Promise(resolve => {
+        resolve(i);
+      });
+    }
+  }
+};
+
+// Usage example:
```

Comment:
Block comments would be more readable and more useful IMO.
```
/**
 * Get time of (latest?) system event
 * @example
 * await dashboardView.sysEventTime(0).getText()
 */
```
See https://github.com/angular/protractor/blob/e77885d7a60fd5a56bf624a5307fd41582f10bca/lib/element.ts#L1073 for examples